### PR TITLE
Add content that translated into zh_Hans

### PR DESCRIPTION
**Pull request by newbe36524**
**Please follow this checklist before submitting:**

- [ x [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Add Content from docs-zh repo as l18n content in zh_Hans

## Issue reference

#1057

### CODE REVIEW

File: daprdocs/config.toml [hunk 1]
```diff
@@ -10,6 +10,18 @@ enableGitInfo = true
 # Language Configuration
 languageCode = "en-us"
 
+[languages]
+  [languages.en]
+    title = "English"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]:
```suggestion
    title = "Dapr Docs"
```

File: daprdocs/config.toml [hunk 2]
```diff
@@ -10,6 +10,18 @@ enableGitInfo = true
...
+    weight = 1
+    contentDir = "content/en"
+    languageName = "English"
+  [languages.zh-hans]
+    title = "简体中文"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]: This should be the translated version "Dapr Docs"

File: .gitmodules [hunk 3]
```diff
@@ -10,3 +10,7 @@
 [submodule "sdkdocs/dotnet"]
 	path = sdkdocs/dotnet
 	url = https://github.com/dapr/dotnet-sdk.git
+[submodule "l18n/docs-zh"]
```
Comment (AaronCrawfis) [inline .gitmodules]:
```suggestion
[submodule "translations/docs-zh"]
```

File: .gitmodules [hunk 4]
```diff
@@ -10,3 +10,7 @@
...
+	path = l18n/docs-zh
```
Comment (AaronCrawfis) [inline .gitmodules]:
```suggestion
	path = translations/docs-zh
```

File: daprdocs/config.toml [hunk 5]
```diff
@@ -37,21 +50,48 @@ id = "UA-149338238-3"
...
     source = "../sdkdocs/dotnet/daprdocs/content/en/dotnet-sdk-docs"
     target = "content/developing-applications/sdks/dotnet"
+    lang = "en"
   [[module.mounts]]
     source = "../sdkdocs/dotnet/daprdocs/content/en/dotnet-sdk-contributing"
     target = "content/contributing/"
+    lang = "en"
+
+  [[module.mounts]]
+    source = "../l18n/docs-zh/content/zh-hans"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]:
```suggestion
    source = "../translations/docs-zh/content/zh-hans"
```

File: daprdocs/config.toml [hunk 6]
```diff
@@ -37,21 +50,48 @@ id = "UA-149338238-3"
...
+    target = "content"
+    lang = "zh-hans"
+  [[module.mounts]]
+    source = "../l18n/docs-zh/content/contributing"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]:
```suggestion
    source = "../translations/docs-zh/content/contributing"
```

File: daprdocs/config.toml [hunk 7]
```diff
@@ -37,21 +50,48 @@ id = "UA-149338238-3"
...
+    target = "content/contributing/"
+    lang = "zh-hans"
+  [[module.mounts]]
+    source = "../l18n/docs-zh/content/sdks_python"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]:
```suggestion
    source = "../translations/docs-zh/content/sdks_python"
```

File: daprdocs/config.toml [hunk 8]
```diff
@@ -37,21 +50,48 @@ id = "UA-149338238-3"
...
+    target = "content/developing-applications/sdks/python"
+    lang = "zh-hans"
+  [[module.mounts]]
+    source = "../l18n/docs-zh/content/sdks_php"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]:
```suggestion
    source = "../translations/docs-zh/content/sdks_php"
```

File: daprdocs/config.toml [hunk 9]
```diff
@@ -37,21 +50,48 @@ id = "UA-149338238-3"
...
+    target = "content/developing-applications/sdks/php"
+    lang = "zh-hans"
+  [[module.mounts]]
+    source = "../l18n/docs-zh/content/sdks_dotnet"
```
Comment (AaronCrawfis) [inline daprdocs/config.toml]:
```suggestion
    source = "../translations/docs-zh/content/sdks_dotnet"
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Thank you for your contribution! Tested it out on my side and functionally everything looks good, just 2 notes on renaming the directory with the submodule and fixing the titles of the pages to say "Dapr Docs" instead of the language name

File: daprdocs/config.toml (hunk 1 above)
Comment (newbe36524) [reply daprdocs/config.toml]: done

File: daprdocs/config.toml (hunk 2 above)
Comment (newbe36524) [reply daprdocs/config.toml]: done

File: .gitmodules (hunk 4 above)
Comment (newbe36524) [reply .gitmodules]: done

**Comment by newbe36524** [comment]
> Thank you for your contribution! Tested it out on my side and functionally everything looks good, just 2 notes on renaming the directory with the submodule and fixing the titles of the pages to say "Dapr Docs" instead of the language name

Thanks, I have updated as your suggestion

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**